server/netty/src/main/java/io/deephaven/server/netty/NettyServerModule.java

```diff
@@ -49,7 +52,11 @@ static GrpcServer serverBuilder(
         interceptors.forEach(serverBuilder::intercept);
         serverBuilder.intercept(MTlsCertificate.DEFAULT_INTERCEPTOR);
         serverBuilder.maxInboundMessageSize(serverConfig.maxInboundMessageSize());
+        Logger log = LoggerFactory.getLogger(NettyServerModule.class);
         if (serverConfig.ssl().isPresent()) {
+            if (log.isInfoEnabled()) {
```

Comment:
while not a bad thing, this seems unused/unrelated to version being runtime configurable?

Reply:
Yes, this was "make version runtime configurable + all other extras from demo branch".

I'd be happy to remove any part of this you don't like, it's just the catch all "here's little things I saw along the way".

Reply:
We *can* do this, but I think we'll want to update the SSLConfig hierarchy to annotate some fields as `org.immutables.value.Value.Redacted`.

Reply:
what needs to be redacted?
I found three password fields, I think I redacted correctly...

Reply:
lgtm